ui/src/components/media/calling.rs

```diff
@@ -490,6 +494,11 @@ fn PendingCallDialog(cx: Scope<PendingCallProps>) -> Element {
             }
         };
     }
+    let alive = use_ref(cx, || Arc::new(AtomicBool::new(false)));
```

Comment:
if the `Arc<AtomicBool>` is inside a `UseRef`, i bet it never gets dropped, which would make your check of `Arc::strong_count` useless. Also, you could just pass around a `UseRef<AtomicBool>` - don't need to wrap it in an `Arc`. 

finally, you probably want to set the variable in the handlers for `with_accept_btn` and `with_deny_btn`

Reply:
unless the dev forgets about that part months later when they need to work on it again.